content/en/docs/reference/glossary/helm-chart.md

```diff
@@ -2,7 +2,7 @@
 title: Helm Chart
 id: helm-chart
 date: 2018-04-12
-full_link: https://helm.sh/docs/developing_charts/
+full_link: https://helm.sh/docs/
```

Comment:
Isn't it suppose to be https://helm.sh/docs/howto/charts_tips_and_tricks/ ?

Reply:
@butuzov  OK. I will update https://helm.sh/docs/topics/charts/